content/docs/getting-started/getting-started-with-ocm/create-component-version.md

```diff
@@ -12,40 +12,45 @@ toc: true
 
 ## Creating and Storing Component Versions
 
-Component Versions are created using a `component-constructor.yaml` file, which is a description file that contains one or multiple components. The file describes the components and their artifacts, resources and sources, metadata in form of labels and references to other components.
+Component Versions are created using a `component-constructor.yaml` file, which is a description file that contains one or multiple components. The file describes the components and their artifacts - resources and sources, metadata in form of labels and references to other components.
 
-Component Versions are locally stored in archives using the Common Transfer Format ([CTF](https://github.com/open-component-model/ocm-spec/blob/main/doc/04-extensions/03-storage-backends/ctf.md)). A CTF archive is also the entity used to transfer components between component repositories. A CTF archive may contain any number of component versions. It may also be pushed to an OCM repository.
+Component Versions are locally stored in archives using the [Common Transfer Format (CTF)](https://github.com/open-component-model/ocm-spec/blob/main/doc/04-extensions/03-storage-backends/ctf.md). A CTF archive may contain any number of component versions and is used to transfer components to and between component repositories.
 
-Note that a CTF archive itself is also an OCM repository, so it can be used as source or target for component transfer operations.
+Note that a CTF archive itself is also an OCM repository, so it can be used as source or target for component transfer operations using the OCM CLI.
 
 The command [`ocm add componentversions`](https://github.com/open-component-model/ocm/blob/main/docs/reference/ocm_add_componentversions.md)
-directly creates a component version from a `component-constructor.yaml` file and stores it in a CTF archive.
+directly creates a component version from a `component-constructor.yaml` file and stores it in a local CTF archive.
 
 ### Create a Component Version
 
 In this examle we will use the The `ocm` CLI tool to create a very basic component version that contains a local resource and a resource that is accessed from a remote location. The local resource is the `podinfo` Helm Chart and the referenced resource is a Docker image stored in an OCI registry.
 
 We start by creating a test folder where we execute all required steps for this example and navigating into it:
 
-```bash
+```shell
```

Comment:
in your last change, you switched it to `bash`, now back to `shell`?!?! Please choose one and stick to it.